split_csv.py

```diff
@@ -0,0 +1,74 @@
+# Split the data into county_year.csv files and check if all rows are exported
+
+import csv
+import os
+import sys
+
+if len(sys.argv) != 2:
+    print('Usage: python split_csv.py giant-file.csv')
+    sys.exit()
+
+input_file = sys.argv[1]
+
+output_dir = 'split_files'
+if not os.path.exists(output_dir):
+    os.makedirs(output_dir)
+    print("Created 'split_files' directory")
+
+county_year_data = {}
+
+# Count total input rows (excluding header)
+print("Counting total input rows")
+with open(input_file, 'r') as infile:
+    total_input_rows = sum(1 for line in infile) - 1
+print(f"Total input rows (excluding header): {total_input_rows}")
+
+print("\nReading and splitting file...")
+with open(input_file, 'r') as infile:
+    reader = csv.reader(infile)
+    headers = next(reader)
+    processed_rows = 0
+    
+    for row in reader:
+        processed_rows += 1
+        if len(row) > 23:
+            # If we have extra columns (which is a consistent error that was found), combine land_use fields
+            row[14] = row[14] + ' ' + row[15]  # Combine land_use fields
+            row = row[:15] + row[16:]  # Remove the extra field
+        
+        county = row[13]
+        year = row[2]
```

Comment:
using the csv reader is there not a way to get the year and county by header name instead of needing to make sure the year is always in column 3?